packages/backend/src/nest/connections-manager/connections-manager.service.ts

```diff
@@ -424,6 +438,7 @@ export class ConnectionsManagerService extends EventEmitter implements OnModuleI
       this.serverIoProvider.io.emit(SocketActionTypes.PEER_DISCONNECTED, payload)
     })
     await this.storageService.init(_peerId)
+    await this.registrationService.init(this.storageService)
```

Comment:
This is not a recommended way to inject dependencies.

Please add StorageModule in Registration Module:
`imports: [StorageModule]`
And in RegistartionService add:
`private readonly storageService: StorageService`